cms/djangoapps/contentstore/tests/test_import.py

```diff
@@ -172,8 +172,8 @@ def test_import_performance_mongo(self):
         # we try to refresh the inheritance tree for each update_item in the import
         with check_exact_number_of_calls(store, 'refresh_cached_metadata_inheritance_tree', 28):
 
-            # _get_cached_metadata_inheritance_tree should be called only once
-            with check_exact_number_of_calls(store, '_get_cached_metadata_inheritance_tree', 1):
+            # _get_cached_metadata_inheritance_tree should be called twice (once for import, once on publish)
+            with check_exact_number_of_calls(store, '_get_cached_metadata_inheritance_tree', 2):
```

Comment:
Is this a valid test change, for `_get_cached_metadata_inheritance_tree` to be called one more time on import?